### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by dev89s**
Added the overlay menu in mobile view:
1. Added the overlay menu and added styling (Background, Font-size, etc.)
2. The menu appears when the hamburger button is clicked
3. The many disappear when the X closing button is clicked or any of the links are clicked.
4. When any link in the menu is clicked it goes to the specific section of the page.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -21,6 +21,48 @@ li {
...
+  background-color: rgba(64, 83, 252, 0.7);
+  backdrop-filter: blur(7px);
+}
+
+.overlay-menu .content {
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+  margin: auto;
+  padding: 20px;
```
Comment (DuaneDave) [inline style.css]: To make the design match the Figma template, I recommend removing this padding property

YOUR DESIGN

uBaDD


FIGMA

baddddyyyy

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @dev89s and @Naledi-Dikgale 👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Harmburger menu is working
✔️ Linters are passing
✔️ PR summary present


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -21,6 +21,47 @@ li {
...
+  width: 80%;
```
Comment (zieeco) [inline style.css:40]: - Hey Team, as pointed out by the previous reviewer, this issue has not yet been resolved. what you should do is to make this width of `80%` to `100%` or better still, remove the width property so it has the default size of 100% taking up the size of its container and the design will be better to fix his changes and to improve your design.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,


Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Mobile menu working :heavy_check_mark: 



## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @dev89s, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**